pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter.go

```diff
@@ -221,6 +221,9 @@ func validateNetworksMatchInterfaces(field *k8sfield.Path, spec *v1.VirtualMachi
 
 	vifMQ = spec.Domain.Devices.NetworkInterfaceMultiQueue
 	isVirtioNicRequested = false
+	if len(spec.Domain.Devices.Interfaces) == 0 {
+		isVirtioNicRequested = true
```

Comment:
There is an api field which is called `autoattachPodInterface`, this is probably the one which you want to check.

Reply:
Thanks, you mean in addition to the current condition right ?
`if len(spec.Domain.Devices.Interfaces) == 0  && *autoattachPodInterface == true`
so we will know that there are no interfaces, and that the default one will be added

Reply:
Yep :+1:

Reply:
should we also check `spec.Networks == 0` ?
this is checked at `setDefaultNetworkInterface`, but a VM without interfaces, but with networks is not admitted

Reply:
I think the network section should be validated in both directions (spec.Networks and spec.Domain.Devices.Interfaces). If we do that right, I think we are good. Right?

Reply:
right, lets be explicit